gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -283,11 +315,26 @@ public GrpcCallContext withStreamWaitTimeout(@Nullable Duration streamWaitTimeou
         endpointContext);
   }
 
+  /**
+   * Backport of {@link #withStreamIdleTimeoutDuration(java.time.Duration)} using {@link
+   * org.threeten.bp.Duration}
```

Comment:
same comment as above - add this to the Javadoc as well

Reply:
Done